crates/router/src/connector/forte/transformers.rs

```diff
@@ -0,0 +1,263 @@
+    pub entered_by: String,
+    #[serde(rename = "billing_address")]
+    pub billing_address: BillingAddress,
+    pub card: Card,
+    pub response: Response,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Response {
+    pub environment: String,
+    #[serde(rename = "response_type")]
+    pub response_type: String,
+    #[serde(rename = "response_code")]
+    pub response_code: String,
+    #[serde(rename = "response_desc")]
+    pub response_desc: String,
+    #[serde(rename = "authorization_code")]
+    pub authorization_code: String,
+    #[serde(rename = "avs_result")]
+    pub avs_result: String,
+    #[serde(rename = "cvv_result")]
+    pub cvv_result: String,
+}
+
+//Res Types end
+
+//TransactionId Types start
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TransactionByIdResponse {
+    #[serde(rename = "transaction_id")]
+    pub transaction_id: String,
+    #[serde(rename = "organization_id")]
+    pub organization_id: String,
+    #[serde(rename = "location_id")]
+    pub location_id: String,
+    pub status: String,
+    pub action: String,
+    #[serde(rename = "authorization_amount")]
+    pub authorization_amount: i64,
+    #[serde(rename = "authorization_code")]
+    pub authorization_code: String,
+    #[serde(rename = "received_date")]
+    pub received_date: String,
+    #[serde(rename = "billing_address")]
+    pub billing_address: BillingAddress,
+    pub card: Card,
+    pub response: Response,
+    pub links: Links,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PhysicalAddress {
+    #[serde(rename = "street_line1")]
+    pub street_line1: String,
+    #[serde(rename = "street_line2")]
+    pub street_line2: String,
+    pub locality: String,
+    pub region: String,
+    pub country: String,
+    #[serde(rename = "postal_code")]
+    pub postal_code: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Links {
+    pub disputes: String,
+    pub settlements: String,
+    #[serde(rename = "self")]
+    pub self_field: String,
+}
+
+//TransactionId Types end
+//Types End
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for FortePaymentsRequest  {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self,Self::Error> {
+
+        match item.request.payment_method_data {
+            api::PaymentMethod::Card(ref ccard) => {
+                let request  =  FortePaymentsRequest{
+                    billing_address : BillingAddress { first_name: ccard.card_holder_name.peek().clone(), last_name: ccard.card_holder_name.peek().clone() },
+                    card: Card {
+                        card_type               : String::from("visa"),
```

Comment:
why is the card type `visa`, what if we provide mastercard, will this payment fail because of card_type